README.md

```diff
@@ -24,6 +24,19 @@ Why?
 - Stop wasting time maintaining a mentors list. This can be generated from the
   projects.
 
+## Usage
+
+To make the website go live (and test your changes out) run:
+
+    $ python -m SimpleHTTPServer 8080
```

Comment:
Adding these two commands before ```python -m SimpleHTTPServer 8080``` will make the instructions more complete, what do you say?
```
$ git clone https://github.com/coala/projects.git
$ cd projects
```